Formula/amqp-cpp.rb

```diff
@@ -0,0 +1,30 @@
+class AmqpCpp < Formula
+  desc "C++ library for communicating with a RabbitMQ message broker"
+  homepage "https://github.com/CopernicaMarketingSoftware/AMQP-CPP"
+
+  stable do
+    url "https://github.com/CopernicaMarketingSoftware/AMQP-CPP.git", revision: "ed2ffd3f03f2bba11f8af68c69ec96ab40a5c4b7"
+    version "2.6.1"
+  end
+
+  head do
```

Comment:
We prefer hash rockets as well, instead of `:`. Won't be relevant here since the `branch` argument can be removed, but just for future reference.